### PR TITLE
validate-chainspecs: check istanbul eips are in the foundation spec

### PR DESCRIPTION
**Pull request by ordian**

### CODE REVIEW

File: scripts/gitlab/validate-chainspecs.sh [hunk 1]
```diff
@@ -15,6 +15,15 @@ done
 for spec in ethcore/res/ethereum/*.json; do
     if ! ./target/release/chainspec "$spec"; then ERR=1; fi
 done
+
+echo "________Mainnet contains Istanbul EIPs________"
+for eip in $(grep --only-matching "eip.*Transition" ethcore/res/ethereum/istanbul_test.json); do
```
Comment (niklasad1) [inline scripts/gitlab/validate-chainspecs.sh]: how deal with `re-enable transitions`?

Comment (ordian) [reply scripts/gitlab/validate-chainspecs.sh]: Add them to the chain spec?
https://github.com/paritytech/parity-ethereum/pull/11228#discussion_r341586118

Comment (niklasad1) [reply scripts/gitlab/validate-chainspecs.sh]: yeah, I guess that is easiest solution even though they are redundant in this case.

File: scripts/gitlab/validate-chainspecs.sh [hunk 2]
```diff
@@ -15,6 +15,15 @@ done
 for spec in ethcore/res/ethereum/*.json; do
     if ! ./target/release/chainspec "$spec"; then ERR=1; fi
 done
+
+echo "________Mainnet contains Istanbul EIPs________"
```
Comment (niklasad1) [inline scripts/gitlab/validate-chainspecs.sh]: Doesn't have to be addressed in this PR but would be good to run this on the other testing network (hard forks) such as Contantinople and so on.

In order to make sure that we are not breaking anything.

Comment (ordian) [reply scripts/gitlab/validate-chainspecs.sh]: good point, but I think since we're running tests with `istanbul_test` spec, it should be caught there

**Comment by TriplEight** [comment]
:+1:

**Pull request merged**